source/javascripts/components/GuidedOnboarding/Collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useRef, useEffect } from "react";
+import { Icon, Text, Flex } from "@bitrise/bitkit";
+import "./Collapsible.scss";
+interface CollapsibleProps {
+  open?: boolean;
+  title: string;
+}
+
+const Collapsible: React.FC<CollapsibleProps> = ({ open = false, children, title }) => {
+  const [isOpen, setIsOpen] = useState(open);
+  const [contentHeight, setContentHeight] = useState<number | undefined>(
+    open ? undefined : 0
+  );
+  const ref = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (isOpen) setContentHeight(ref.current?.getBoundingClientRect().height);
+    else setContentHeight(0);
+  }, [isOpen]);
+
+  const toggleOpen = () => {
+    setIsOpen((prev) => !prev);
+  };
+
+  return (
+      <Flex
+        className="guided-onboarding-collapsible"
+        direction='vertical'
+    >
+        <Flex className="title" direction='horizontal' alignChildrenVertical='middle' alignChildrenHorizontal='between'
+        >
+            <Flex direction='horizontal' gap='x2' alignChildrenVertical='middle'>
+                <Icon name='Info' size='24px' />
+                <Text size='3' weight='bold' style={{ lineHeight: "16px" }}>{title}</Text>
```

Comment:
Again, I'm not sure of what's currently available in WFE, but we generally want to steer clear of inline styles. Luckily, the latest available design system fonts have the correct line heights and letter spacing baked right into the size values, so we should be able to avoid doing this after the BitKit upgrade.